collections/_article/dev-snapshot-godot-4-2-dev-4.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 4"
+excerpt: "Progressing steadily towards the beta stage, the fourth dev snapshot brings a lot of improvements all around."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-4.webp
+image_caption_title: "AssetPlacer"
+image_caption_description: "A plugin by CookieBadger"
+date: 2023-09-05 16:00:00
+---
+
+We're making great progress in the development branch for Godot 4.2! The faster paced release cycle we started following after the 4.0 release seems to be working well, with pull requests being opened, reviewed, and merged at a steady state. This fourth dev snapshot contains nearly 250 PRs merged in a little over 3 weeks since the [dev 3 snapshot](/article/dev-snapshot-godot-4-2-dev-3/).
+
+We're slightly behind our internal release schedule on that build, but only because we had some unplanned maintenance of the build server. Otherwise the overall state of the `master` branch seems pretty good and on track for reaching the beta stage at the start of October, and stable in early November 2023.
```

Comment:
I'm not sure which build is "that build" - dev 3 or dev 4. In any case, is it important to mention the slight delay? It would seem clearer just to say e.g.:
```suggestion
The overall state of the `master` branch seems pretty good and on track for reaching the beta stage at the start of October, and stable in early November 2023.
```

Reply:
It referred to dev4. It's not necessary to mention, but I like to give users some insights into how we work. Can remove though.

Reply:
I suggest changing "that build" to "this build" or even "this dev 4 build" to make it clear.